module/VuFind/tests/unit-tests/src/VuFindTest/Controller/Plugin/ResultScrollerTest.php

```diff
@@ -412,18 +418,20 @@ class ResultScrollerMock extends \VuFind\Controller\Plugin\ResultScroller
      */
     protected $testResults;
 
-    public function __construct($mockManager, $testResults)
+    public function __construct($mockManager, $testResults, $memory)
```

Comment:
Is there a compelling reason to give the mock class a different constructor signature than its parent? Would it be better to match the signature and hard-code the true up above where we construct this instead?

Reply:
It was already different; the real ResultScroller doesn't have the $testResults parameter. But I dropped the constructor and added a new method for setting `$testResults`.